hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/security/symmetric/TestSecretKeyManager.java

```diff
@@ -110,16 +110,15 @@ public void testLoadSecretKeys(List<ManagedSecretKey> savedSecretKey,
       // expect the current key is newly generated.
       assertFalse(savedSecretKey.contains(state.getCurrentKey()));
```

Comment:
This one can also be converted to `assertThat`.

Reply:
fixed